### ISSUE TITLE
Update/Clarification to Readme

### ISSUE THREAD

**Issue by gctseng** [syscl/XPS9350-macOS#16]
@syscl I realize your instructions in the Readme are supposed to be an "overview" for people who are familiar with Clover and the whole process, but I wanted to just add a couple of extra steps to help others who are not as familiar.  Also, capturing the ACPI tables in step 5 (before the install even happens) will allow the original tables to be copied over to the SSD (you didn't mention this in your current Readme so when I ran deploy.sh the first time it couldn't find the tables and I rebooted--before installing Clover on the SSD and copying your CLOVER folder over, with that whole issue of the USB not being recognized anymore, etc., haha) and saving the user a reboot.

I haven't had a chance to learn how to use git to add suggested changes to the files, but in the meantime, see below: 

1.     Create a vanilla installation disk (e.g., USB or other removable disk).
2.     Install Clover with UEFI only and UEFI64Drivers to the installation disk just created.
3.     Replace the original CLOVER folder with the one under my Git/XPS9350-macOS/CLOVER.
4.     Change BIOS settings (based on "Bios Settings" in post 1 on this thread: https://www.tonymacx86.com/threads/guide-dell-xps-13-9350-macos-10-12-1.204730/ )
- Disable Secure Boot
- Disable VT for Direct I/O
- Set Auto OS Recovery Threshold to OFF
- Set SATA Operation to AHCI or Disabled
- Optional: Disable Computrace
- Optional: Enable Unobtrusive Mode
- Optional: Enable USB PowerShare
- Optional: Set Fn Lock Options to Lock Mode Enable/Secondary
5.     Boot to USB and hit F4 and/or Fn-F4 to capture ACPI tables to the USB
6.     Install  macOS but do not reboot until you have finished steps 7 and 8, below.
7.      Install Clover with UEFI only and UEFI64Drivers to SSD.
8.     Replace the original CLOVER folder on the SSD with the one from the USB drive under my Git/XPS9350-macOS/CLOVER.
9.     Once you finish installation of macOS, you can do the following steps to finish the post installation of macOS

**Comment by syscl** [comment]
Thanks for the detail information. If you can fork this project then refine the readme I will be more appreciate. 

syscl

**Comment by gctseng** [comment]
Haha, give me a few days to figure git out.  Haven't gotten beyond the downloading/refreshing of a repository yet.

**Comment by nos1609** [comment]
@gctseng what for are these two points?
I'm running hackintosh for two months with these options enabled and everyting is just perfect. The only changes really needed is secureboot and SATA only.

> Disable VT for Direct I/O
Set Auto OS Recovery Threshold to OFF

**Comment by syscl** [comment]
@nos1609 Right, that's why I just point out change AHCI... No other settings need to change...

syscl

**Comment by gctseng** [comment]
@nos1609 @syscl I just followed the other guide.  On my upcoming reinstall I will just set the AHCI.  That is, I will reset the bios to defaults and just set the AHCI (and turn off secure boot, of course).

**Comment by Eduardoveras94** [comment]
@gctseng if you want i can open a pull request with the readme updated, or are you waiting for something?

**Comment by lorenzolamasse** [comment]
Dear all,

A few comments:
- lacking some details in step 2, which 64UEFI drivers are to be selected: only OsxAptioFixDrv-64 or any other driver ? Or maybe it has no interest as they are overwritten at next step, so this whole point is useless except UEFIOnly selection
- when creating the USB, standard USB creation (createinstallmedia) does not work, only "asr restore" method (at least at my place, please confirm if you encounter the same behavior) (from https://www.tonymacx86.com/threads/guide-booting-the-os-x-installer-on-laptops-with-clover.148093/)
- also, rebooting is automatic after installation so there is no way to install Clover before restart
